packages/ui/src/components/AddGroupSheet.tsx

```diff
@@ -0,0 +1,373 @@
+            duration: 300,
+            easing: Easing.out(Easing.cubic),
+          }),
+        },
+      ],
+      opacity: withTiming(isActive ? 1 : 0, {
+        duration: 300,
+        easing: Easing.out(Easing.cubic),
+      }),
+    };
+  }, [isActive]);
+
+  return (
+    <Animated.View style={[styles.screen, animatedStyles]}>
+      {children}
+    </Animated.View>
+  );
+};
+
+const styles = StyleSheet.create({
+  screen: {
+    ...StyleSheet.absoluteFillObject,
+  },
+});
+
+export function AddGroupSheet({
+  open,
+  onOpenChange,
+  onCreatedGroup,
+  onGoToDm,
+  navigateToFindGroups,
+}: {
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+  onCreatedGroup: ({
+    group,
+    channel,
+  }: {
+    group: db.Group;
+    channel: db.Channel;
+  }) => void;
+  onGoToDm: (userId: string) => void;
+  navigateToFindGroups: () => void;
+}) {
+  const [screenScrolling, setScreenScrolling] = useState(false);
+  const [screenKey, setScreenKey] = useState<number>(0);
+  const [currentScreen, setCurrentScreen] = useState<CurrentScreenKey>('Root');
+  const [groupInvitees, setGroupInvitees] = useState<string[]>([]);
+  const contacts = useContacts();
+
+  const dismiss = useCallback(() => {
+    onOpenChange(false);
+    // used for resetting components nested within screens after
+    // reopening
+    setTimeout(() => {
+      setCurrentScreen('Root');
+      setScreenKey((key) => key + 1);
+    }, 300);
+  }, [onOpenChange]);
+
+  useEffect(() => {
+    if (open) {
+      triggerHaptic('sheetOpen');
+    }
+  }, [open]);
+
+  const goToScreen = useCallback(
+    (screen: CurrentScreenKey) => {
+      setCurrentScreen(screen);
+    },
+    [setCurrentScreen]
+  );
+
+  const screens: Record<CurrentScreenKey, React.ReactElement> = useMemo(
+    () => ({
+      Root: (
+        <TransitionWrapper key={'Root'} isActive={currentScreen === 'Root'}>
+          <RootScreen
+            goToScreen={goToScreen}
+            goToDM={onGoToDm}
+            goToFindGroups={navigateToFindGroups}
+          />
+        </TransitionWrapper>
+      ),
+      CreateGroup: (
+        <TransitionWrapper
+          key={'CreateGroup'}
+          isActive={currentScreen === 'CreateGroup'}
+        >
+          <CreateGroupScreen invitees={groupInvitees} goToScreen={goToScreen} />
+        </TransitionWrapper>
+      ),
+      InviteUsers: (
+        <TransitionWrapper
+          key={'InviteUsers'}
+          isActive={currentScreen === 'InviteUsers'}
+        >
+          <InviteUsersScreen
+            invitees={groupInvitees}
+            setInvitees={setGroupInvitees}
+            goToScreen={goToScreen}
+          />
+        </TransitionWrapper>
+      ),
+    }),
+    [goToScreen, onGoToDm, groupInvitees, currentScreen, navigateToFindGroups]
+  );
+
+  return (
+    <ActionSheet
+      disableDrag={screenScrolling}
+      moveOnKeyboardChange
+      open={open}
+      onOpenChange={dismiss}
+      snapPoints={['90%']}
```

Comment:
This fixes the snapPoints warning for me

```suggestion
      snapPoints={[90]}
      snapPointsMode="percent"
```